postgresql-16.yaml

```diff
@@ -190,47 +190,6 @@ subpackages:
           ln -s /usr/libexec/${{vars.mangled-package-name}}/docker-ensure-initdb.sh ${{targets.subpkgdir}}/usr/bin/docker-ensure-initdb.sh
           ln -s /usr/libexec/${{vars.mangled-package-name}}/docker-ensure-initdb.sh ${{targets.subpkgdir}}/var/lib/postgres/initdb/docker-ensure-initdb.sh
 
```

Comment:
This package should also be bumped epoch to 1.

Such that there is no set of melange files that build the same full-version that build the same .apk.